nextcord/guild_onboarding.py

```diff
@@ -0,0 +1,139 @@
+# SPDX-License-Identifier: MIT
```

Comment:
While you make these changes you could add versionadded to these three, don't worry if not though.